scripts/prod_stag_sanity_check.py

```diff
@@ -0,0 +1,72 @@
+import json
+from math import isclose
+import os
+import random
+
+import boto3
+from helpers.enums import Network
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def download_boosts(chain: str, bucket: str):
+    s3 = boto3.client(
+        "s3",
+        aws_access_key_id=os.environ["AWS_ACCESS_KEY_ID"],
+        aws_secret_access_key=os.environ["AWS_SECRET_ACCESS_KEY"],
+    )
+
+    print("DOWNLOADING BOOST")
+    chain_ids = {
+        Network.Ethereum: 1,
+        Network.Arbitrum: 42161,
+        Network.Polygon: 137,
+    }
+    chain_id = chain_ids[chain]
+
+    boost_file_name = f"badger-boosts-{chain_id}.json"
+    s3ClientObj = s3.get_object(Bucket=bucket, Key=boost_file_name)
+    data = json.loads(s3ClientObj["Body"].read().decode("utf-8"))
+    return data
+
+
+supported_chains = [Network.Ethereum, Network.Arbitrum, Network.Polygon]
+for chain in supported_chains:
+
+    prod_boosts = download_boosts(chain, "badger-merkle-proofs")
```

Comment:
can we store these in vars in constants like STAGING_MERKLE_BUCKET or something? then can refactor other code references to use them